### PR TITLE
Willes/solana contract call rpc implementation

### PR DESCRIPTION
**Pull request by willeslau**

### CODE REVIEW

File: common/Cargo.toml [hunk 1]
```diff
@@ -6,3 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+borsh = "0.9"
```
Comment (ivanceras) [inline common/Cargo.toml:9]: nice, NEAR also uses borsh. At first, Borsh, sounds like a scientist name, but there's really nothing special in comparison to serde.
Borsh stands for Binary Object Representation Serializer for Hashing.

File: solana/beacon-server/tests/beacon-server.ts [hunk 2]
```diff
@@ -1,16 +1,89 @@
...
+  //   const tx = await program.rpc.updateDapiWithBeacons(
+  //     tempDAPIId,
+  //     [beaconID],
+  //     {
+  //       accounts: {
+  //         dapi: dapiPDA,
+  //         user: anchor.getProvider().wallet.publicKey,
+  //         systemProgram: anchor.web3.SystemProgram.programId,
+  //       },
+  //       remainingAccounts: [
```
Comment (ivanceras) [inline solana/beacon-server/tests/beacon-server.ts:79]: perhaps remove this commented out code here?

**Review by ivanceras — CHANGES_REQUESTED** [review]
just some nitpicks

File: common/Cargo.toml (hunk 1 above)
Comment (willeslau) [reply common/Cargo.toml:9]: haha, sounds like a scientist indeed, or a car brand

File: solana/beacon-server/tests/beacon-server.ts (hunk 2 above)
Comment (willeslau) [reply solana/beacon-server/tests/beacon-server.ts:79]: I will reenable them, forgot the uncomment

**Review by ivanceras — APPROVED** [review]

**Pull request merged**